src/Tokenizer/Tokens.php

```diff
@@ -572,6 +584,43 @@ public function getNonWhitespaceSibling(int $index, int $direction, ?string $whi
         }
     }
 
+    /**
+     * Get index for closest sibling token which is non meta (whitespace, comment, PHP8 attribute).
```

Comment:
curious - is `meta` some formal naming here, or only a word you picked? (happy with both, just curious)